### PR TITLE
Feature/changes from first feedback

### PR DESCRIPTION
**Pull request by ChrisSargent**

### CODE REVIEW

File: src/styles/_utils/sizes.scss [hunk 1]
```diff
@@ -1,11 +1,11 @@
 @import 'functions';
 
-$cn-narrow: 600px;
+$cn-narrow: 1100px;
```
Comment (ChrisSargent) [inline src/styles/_utils/sizes.scss]: Why is this changed?

File: src/styles/hero-with-title.scss [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
   width: 100%;
-  height: $height-hero;
   overflow: hidden;
   background-repeat: no-repeat;
   background-position: 50%;
   background-size: cover;
 
+  &::after {
+    display: block;
+    padding-bottom: $height-hero;
```
Comment (ChrisSargent) [inline src/styles/hero-with-title.scss]: Since this is now a 'ratio', I think should be renamed?

File: src/styles/slider.scss [hunk 3]
```diff
@@ -1,16 +1,20 @@
 @import 'lightslider/src/css/lightslider';
 
 .slider {
-  height: $height-hero;
   // Ensures no flash of images or artifacts whilst slider js is loading
```
Comment (ChrisSargent) [inline src/styles/slider.scss]: Because this no longer has a fixed height, there is a problem with the images flashing up on first page load.

File: src/styles/_utils/sizes.scss (hunk 1 above)
Comment (prikhoda-natalia) [reply src/styles/_utils/sizes.scss]: I needed a narrow container for section that wraps Instagram feed - it made sense to use this variable `$cn-narrow`. I searched where else it was used and found only one place - as `max-width` for paragraph in article-card component. My thinking was that container's width and article-card paragraph's width are 2 different things - so I updated value for this variable and changed styles in article-card.

Comment (ChrisSargent) [reply src/styles/_utils/sizes.scss]: Okay - but the paragraph on the first page is now much too wide. So maybe it needs a max-width prop too?

File: src/styles/slider.scss (hunk 3 above)
Comment (prikhoda-natalia) [reply src/styles/slider.scss]: changed name to `$height-hero-ratio`

File: src/styles/hero-with-title.scss (hunk 2 above)
Comment (prikhoda-natalia) [reply src/styles/hero-with-title.scss]: changed name to $height-hero-ratio

File: src/styles/slider.scss (hunk 3 above)
Comment (prikhoda-natalia) [reply src/styles/slider.scss]: fixed the problem in new commit

File: src/styles/_utils/sizes.scss (hunk 1 above)
Comment (prikhoda-natalia) [reply src/styles/_utils/sizes.scss]: added new container width - `xs` - for the paragraph on the first page
